### PR TITLE
ENG-2563 Refactored users-management tests

### PR DESCRIPTION
**Pull request by g-di-martino**
Updated intelliJ config files
Created sections folder structure in integration folder
Created DialogContent class to handle different Dialogs
Updated table row validation to make it consistent
Renamed login example to keyCloak.spec and added more validations
Refactored users-management tests

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -1,7 +1,12 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
     <JSCodeStyleSettings version="0">
```
Comment (gReis89) [inline .idea/codeStyles/Project.xml:3]: hummm I think we should configure our eslint to try matching these styles rules to keep the code pattern between IntelliJ and VSCode devs

Comment (g-di-martino) [reply .idea/codeStyles/Project.xml:3]: I thought everyone was using IntelliJ 😅 
I can have a look and configure eslint as well, but I would keep IntelliJ configurations as those are better integrated with the IDE and it is quicker to apply a fix if you fail following it.
For some of the configuration, we could also use [editorconfig](https://editorconfig.org) (which I already introduced), but it requires a plugin to work with VSCode

File: ui-tests/cypress/support/pageObjects/users/management/ManagementPage.js [hunk 2]
```diff
@@ -1,5 +1,167 @@
...
+  }
+
+  openAddUserPage() {
+    this.getAddButton().click();
+    return new AppPage(AddPage);
+  }
+
+}
+
+class UsersKebabMenu extends KebabMenu {
```
Comment (gReis89) [inline ui-tests/cypress/support/pageObjects/users/management/ManagementPage.js:116]: it doesn't worth moving this class to its own file?

**Review by gReis89 — APPROVED** [review]
LGTM! needs a rebase!

**Pull request merged**